hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -1036,14 +1037,45 @@ public boolean isEmpty() {
 
     @Override
     public void putAll(Map<? extends K, ? extends V> m) {
```

Comment:
Comments for hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java putAll apply to this method as well.